<?xml version="1.0"?>
<launch>
  <node pkg="frenet_planner" type="frenet_planner_node" name="frenet_planner_node">
    <param name="max_speed" value="5.3" /> <!--km/h-->
    <param name="max_accel" value="2.0" /> <!--m/ss-->
    <param name="max_curvature" value="0.5" /> <!--1/m-->
    <param name="max_road_width" value="1.5" />
    <param name="d_road_w" value="0.10" />
    <param name="maxT" value="7.0" />
    <param name="minT" value="4.0" />
    <param name="dT" value="1.5" />
    <param name="d_t_s" value="0.10" /> <!--目标速度采样间隔 [m/s]-->
    <param name="n_s_sample" value="0" /> <!--sampling number -->
    <param name="robot_radius" value="0.30" />

    <param name="KJ"  value="0.1" />
    <param name="KT"  value="0.1" />
    <param name="KD"  value="2.0" />
    <param name="KLAT" value="3.5" />
    <param name="KLON" value="0.5" />
    <param name="KCON" value="0.5" />

    <param name="patch_width" value="20" />
    <param name="patch_height" value="40" />
    <param name="d_width" value="0.10" />
    <param name="d_height" value="0.25" />
  </node>
</launch>
